Train Begin...
Episodes:   0%|                                                                   | 0/1000 [00:00<?, ?it/s]
Traceback (most recent call last):                                                                         
경로 재계산 실패: AGV F, 현재 위치 (50, 12), 시작 위치 (40, 3), 목표 위치 (40, 3)
경로 재계산 실패: AGV O, 현재 위치 (70, 12), 시작 위치 (60, 3), 목표 위치 (60, 3)
  File "/home/js/DAA_CPS/train.py", line 109, in <module>
    train(config)
  File "/home/js/DAA_CPS/train.py", line 59, in train
    action = agent.act(state, env, agv_id)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/js/DAA_CPS/Agent.py", line 82, in act
    env.controller.perform_action(agv_id, action)
  File "/home/js/DAA_CPS/Controller.py", line 139, in perform_action
    self.replan_rout(agv_id)
  File "/home/js/DAA_CPS/Controller.py", line 147, in replan_rout
    current_goal = self.agv_rout[agv_id][-1]
                   ~~~~~~~~~~~~~~~~~~~~~^^^^
IndexError: list index out of range
